src/devtools/client/debugger/src/components/TestInfo/TestSteps.tsx

```diff
@@ -155,26 +169,33 @@ function TestSection({
     <>
       {header ? <div className="py-2">{header}</div> : null}
       {steps.map((s, i) => (
```

Comment:
Another way to approach this would be to merge and sort steps and events in one array and then render each in one array pass